sdk/metric/sdk.go

```diff
@@ -218,7 +218,9 @@ func (s *syncInstrument) acquireHandle(kvs []attribute.KeyValue, labelPtr *attri
 		// needed for the `sortSlice` field, to avoid an
 		// allocation while sorting.
 		rec = &record{}
-		rec.storage = attribute.NewSetWithSortable(kvs, &rec.sortSlice)
+		tmpKvs := make([]attribute.KeyValue, len(kvs))
+		copy(tmpKvs, kvs)
```

Comment:
Please add a comment here explaining why the input is copied so that someone who comes along later and thinks they'll "optimize" it might think twice! :)

Reply:
comments added